modules/PrimaryPool/substreams.yaml

```diff
@@ -27,13 +34,27 @@ modules:
     kind: map
     initialBlock: 8586768
     inputs:
+      - source: sf.ethereum.type.v2.Block
+      # Added UP
       - map: map_pools
     output:
-      type: proto:verified.primary.v1.Subscriptions
+      type: proto:verified.primary.v1.PrimaryIssues
+
+  - name: map_subscriptionsCheck
+    kind: map
+    initialBlock: 8586768
+    inputs:
+      - source: sf.ethereum.type.v2.Block
+    output:
+      type: proto:verified.primary.v1.PrimaryIssues
 
   - name: kv_out
     kind: map
     inputs:
-      - map: map_subscriptions
+      - store: store_block_meta_end
```

Comment:
@yaksheshgupta should not this input be the subscriptions map ?

Reply:
Yes, it should be.
I just commented out the code for testing purpose only